--- src/rviz/src/rviz/mesh_loader.cpp   2016-06-11 17:55:38.815197313 +0000
+++ src/rviz/src/rviz/mesh_loader.cpp   2016-06-11 17:59:25.503550955 +0000
@@ -67,6 +67,15 @@
 #include <assimp/IOSystem.h>
 #endif
 
+#  ifdef __arm__                 // fix for ARM build
+// from http://answers.ros.org/question/52098/unable-to-compile-rviz-on-ubuntu-armhf/
+#include <strings.h>
+bool Assimp::IOSystem::ComparePaths(const char *p1, const char *p2) const
+{
+    return !::strcasecmp(p1, p2);
+}
+#  endif
+
 namespace fs = boost::filesystem;
 
 namespace rviz
